From 742687ee7bf4599276e1b9722645ea1afc4d34ff Mon Sep 17 00:00:00 2001
From: Teijo Kinnunen <teijo.kinnunen@uros.com>
Date: Fri, 4 Sep 2020 09:24:39 +0300
Subject: [PATCH 5/5] collections,basic: add release USSD indication

This indication is used by MM and omitting it causes crashes in MM.

(cherry picked from commit 87f35a898ecc14b3fd80d9453ed010725b11835f)
---
 data/qmi-collection-basic.json | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/data/qmi-collection-basic.json b/data/qmi-collection-basic.json
index 617b151..e56e7b3 100644
--- a/data/qmi-collection-basic.json
+++ b/data/qmi-collection-basic.json
@@ -150,5 +150,6 @@
     "QMI_MESSAGE_VOICE_ORIGINATE_USSD",
     "QMI_MESSAGE_VOICE_ANSWER_USSD",
     "QMI_MESSAGE_VOICE_CANCEL_USSD",
-    "QMI_INDICATION_VOICE_USSD"
+    "QMI_INDICATION_VOICE_USSD",
+    "QMI_INDICATION_VOICE_RELEASE_USSD"
 ]
-- 
2.29.2

